docusaurus/docs/dev-docs/api/plugins/admin-panel-api.md

```diff
@@ -177,14 +145,8 @@ The Admin Panel API allows a plugin to take advantage of several small APIs to p
 | Inject a Component in an injection zone  | [Injection Zones API](#injection-zones-api) | [`injectComponent()`](#injection-zones-api)           | [`bootstrap()`](#register)  |
 | Register a hook                          | [Hooks API](#hooks-api)                 | [`registerHook()`](#hooks-api)                    | [`bootstrap()`](#bootstrap)   |
 
-:::tip Replacing the WYSIWYG
-The WYSIWYG editor can be replaced by taking advantage of [custom fields](/dev-docs/custom-fields), for instance using the [CKEditor custom field plugin](https://market.strapi.io/plugins/@ckeditor-strapi-plugin-ckeditor).
-:::
-
-:::info
```

Comment:
Why was it removed? Is it wrong?
It was added through a community PR a while ago because users were missing this piece of information and we thought this would be the appropriate place to mention it.

Reply:
I have removed it because this part of the docs is about "Available actions". While the Editor is using some of them I think we should keep it out of these reference docs, because it interrupts the reading flow with something (IMO) unrelated.

Reply:
Callouts are actually meant to be skipped for users who don't need the information :) 

It was added because some users were looking to achieve such an action while reading this API documentation.
We can try to remove it, but if users are lost or ask about it, I'll reintroduce it.

I definitely think cross-referencing sections help users find what they're looking for (a bit like you could find a potato masher next to potatoes in some supermarkets :D)